src/test/java/com/wikia/webdriver/testcases/wampagetests/WamPageTests.java

```diff
@@ -11,11 +11,6 @@
 
 import java.util.EnumSet;
 
-/**
- * @author Qaga
- * @ownership Content X-Wing
```

Comment:
These test are veryfing UI elements. I assume it's our for Oasis (Skin) component
will ask po